### PR TITLE
convert_bg missing in many statements for rT

### PR DESCRIPTION
**Pull request by mountrcg**
I changed many rT statements for BG's to include `convert_bg function`. However for the `rT.predBGs` in the for-Loop that doesn't work. I would require some help there.

### CODE REVIEW

File: lib/determine-basal/determine-basal.js [hunk 1]
```diff
@@ -695,7 +695,7 @@ var determine_basal = function determine_basal(glucose_status, currenttemp, iob_
         }
 
         // set eventualBG based on COB or UAM predBGs
-        rT.eventualBG = eventualBG;
+        rT.eventualBG = convert_bg(eventualBG,profile);  // for FAX needs to be in mg/dL
```
Comment (scottleibrand) [inline lib/determine-basal/determine-basal.js]: What is FAX (edit: FreeAPSX?)?

Are you intending to comment out this line in light of #1439? Looks like it's still uncommented.

Comment (scottleibrand) [reply lib/determine-basal/determine-basal.js]: If you want to comment out / revert this one for now until we figure out #1439, the other console.error lines are perfectly safe to change, and I can merge this PR with just those.

Comment (mountrcg) [reply lib/determine-basal/determine-basal.js]: yes FAX = FreeAPS-X, there the rT variables are used to populate the swift code

Comment (mountrcg) [reply lib/determine-basal/determine-basal.js]: do you have an idea how to convert the prediction values UAM, ZT and the like as well?

Comment (mountrcg) [reply lib/determine-basal/determine-basal.js]: > if you want to comment out / revert this one for now

ok, forgot about it - yes, I took it out now

**Review by scottleibrand — APPROVED** [review]

**Comment by scottleibrand** [comment]
Thx. All changes are now cosmetic, and should only affect mmol/L users, so merging.

**Comment by sulkaharo** [comment]
Was this tested with Nightscout btw? 😅

**Comment by scottleibrand** [comment]
@sulkaharo which changes? AFAICT this only affects console.error logging, not anything uploaded.

**Comment by sulkaharo** [comment]
Ah right, misread the code and assumed some data is also uploaded

**Pull request merged**